### ISSUE TITLE
"The specified framework 'Microsoft.WindowsDesktop.App' is not present in the previously loaded runtime"

### ISSUE THREAD

**Issue by delfuria** [unoplatform/studio#45]
### Current behavior

I'm a HotDesign beta user 
![image](https://github.com/user-attachments/assets/f093220f-473e-4a47-96cf-df6f0553fd6e)
but when i run the App (in Desktop) i get this error message 
![image](https://github.com/user-attachments/assets/5bcf22bc-289e-4f4c-a3d0-0d6d4568f258)
Also there's no "Hot Design" icon 
![image](https://github.com/user-attachments/assets/7e8eb33e-3b45-4afa-979d-aeb0745b000b)

### Expected behavior

"Hot-Design" icon showed correctly

### Which tool(s) are affected?

Hot Design™

### How to reproduce it (as minimally and precisely as possible)

Follow instruction from Counter's tutorial

### Affected platforms

Desktop (Skia)

### Uno.Sdk version

5.5.56

### IDE used

Visual Studio 2022

### IDE version

17.12.3

### Uno Platform Extension version

5.5.61.107

**Comment by DevTKSS** [comment]
This initial message also in issue caption is Normal for uno debugging seen it already mentioned and reason explained but not remembering which uno repo this was. The warnings you get and that icon not showing is in my opinion already listed in an issue to that seen it around here to be discussed

**Comment by nickrandolph** [comment]
@delfuria the UX there is confusing (will be updated in a coming release) - Hot Design is dimmed out which indicates the feature isn't currently enabled.
cc @sasakrsmanovic are you able to add @delfuria to the beta?

**Comment by sasakrsmanovic** [comment]
Hello @delfuria . Confirmed, you now have Hot Design license. Please try refreshing via your Uno Settings app; or sign-out/in. Please let us know if that worked.

**Comment by delfuria** [comment]
Hi, niw the Hot Design button works fine.
I dont' know if this is normal but i get on the debug window a lot of error as showed below
![image](https://github.com/user-attachments/assets/5152e514-ab16-42a3-b3f1-21feb3793e48)

**Comment by nickrandolph** [comment]
@delfuria yes we're aware of the output and will be addressing these in a subsequent update to Hot Design
